functional.py

```diff
@@ -0,0 +1,124 @@
+def sequential_map(*args):
+    """
+    The function implements sequentially any number of functions on the container
+    :param args:
+    :return con:
+    """
+    funcs = args[:-1]
+    container = args[-1]
```

Comment:
Можно вот так вот, супер удобно
```suggestion
    *funcs, container = args
```